src/components/tags/tags.js

```diff
@@ -0,0 +1,101 @@
+import template from "./tags.html";
+import tagItemTemplate from "./tag-item.html";
+import { ButtonMoreComponent } from "../button-more/button-more";
+import { TagListService } from "../../services/tag-service";
+import { AddTagDialogComponent } from "../add-tag-dialog/add-tag-dialog";
+
+export class TagsComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+  }
+
+  querySelectors() {
+    this.tagsRoot = this.mountPoint.querySelector(".tags");
+    this.addTagButton = this.mountPoint.querySelector(
+      ".tags__add-tag-dialog-activation"
+    );
+    this.addTagMountPoint = document.querySelector(".tags__add-tag-dialog");
+    this.tagsList = this.mountPoint.querySelector(".tags__list-items");
+  }
+
+  handleAddTagConfirmed(tag) {
+    TagListService.add(tag);
+    this.addTagToHead(tag);
```

Comment:
you do not init 3dot button after adding tag

![image](https://user-images.githubusercontent.com/36405386/37766208-edb36e14-2dce-11e8-9759-f51eb2d2c0c2.png)